charts/stable/tautulli/templates/secret-ldapsync.yaml

```diff
@@ -0,0 +1,11 @@
+{{- if and (.Values.ldapSync.enabled) (not .Values.ldapSync.existinSecret) }}
```

Comment:
Mistake? this is in the tautulli directory

Reply:
Not sure how that happened.  Corrected